WalletWasabi.Tests/UnitTests/BitcoinCore/RpcBasedTests.cs

```diff
@@ -48,5 +48,39 @@ public async Task AllFeeEstimateAsync()
 				await coreNode.TryStopAsync();
 			}
 		}
+
+		[Fact]
+		public async Task FetchBlockFilterIndexFromCoreAsync()
+		{
+			using var services = new HostedServices();
+			var coreNode = await TestNodeBuilder.CreateAsync(services, blockFilterIndex: "basic");
+			await services.StartAllAsync(CancellationToken.None);
+			try
+			{
+				var rpc = coreNode.RpcClient;
+				var blockFilter = await rpc.GetBlockFilterAsync(Network.RegTest.GenesisHash);
+
+				var genesis = Network.RegTest.GetGenesis();
+				var genesisScriptPubKey = genesis.Transactions[0].Outputs[0].ScriptPubKey;
+				Assert.True(blockFilter.Filter.Match(genesisScriptPubKey.ToBytes(), Network.RegTest.GenesisHash.ToBytes()));
+				Assert.False(blockFilter.Filter.Match(genesisScriptPubKey.ToBytes().Skip(1).ToArray(), Network.RegTest.GenesisHash.ToBytes()));
+
+				// Compare the ScriptPubKey with filters generated by wassabi params.
```

Comment:
```suggestion
				// Compare the ScriptPubKey with filters generated by wasabi params.
```

Reply:
argh I have this obsession with double ssing the wasabi s 😄